### ISSUE TITLE
`SQLite.table` always gives error message.

### ISSUE THREAD

**Issue by AquaIndigo** [JuliaDatabases/SQLite.jl#258]
```julia
julia> SQLite.tables(db)
(name = Union{Missing, String}[],)

julia> SQLite.createtable!(db, "dialogs", Tables.Schema([:dialogID], [Int]))
101

julia> SQLite.tables(db)
ERROR: MethodError: promote_type(::Type{Union{}}, ::Type{String}) is ambiguous. Candidates:
  promote_type(::Type{Union{}}, ::Type{T}) where T in Base at promotion.jl:224
  promote_type(::Type{T}, ::Type{String}) where T<:WeakRefStrings.InlineString in WeakRefStrings at /home/suyao/.julia/packages/WeakRefStrings/a3jYm/src/inlinestrings.jl:44
Possible fix, define
  promote_type(::Type{Union{}}, ::Type{String})
Stacktrace:
  [1] add_or_widen!(val::String, col::Int64, nm::Symbol, dest::Tables.EmptyVector, row::Int64, updated::Base.RefValue{Any}, L::Base.SizeUnknown)
    @ Tables ~/.julia/packages/Tables/E99hk/src/fallbacks.jl:150
  [2] eachcolumns
    @ ~/.julia/packages/Tables/E99hk/src/utils.jl:127 [inlined]
  [3] _buildcolumns(rowitr::SQLite.Query, row::SQLite.Row, st::Nothing, sch::Tables.Schema{(:name,), nothing}, columns::Tuple{Tables.EmptyVector}, updated::Base.RefValue{Any})
    @ Tables ~/.julia/packages/Tables/E99hk/src/fallbacks.jl:187
  [4] buildcolumns
    @ ~/.julia/packages/Tables/E99hk/src/fallbacks.jl:217 [inlined]
  [5] columns
    @ ~/.julia/packages/Tables/E99hk/src/fallbacks.jl:253 [inlined]
  [6] columntable(itr::SQLite.Query)
    @ Tables ~/.julia/packages/Tables/E99hk/src/namedtuples.jl:171
  [7] execute(f::typeof(Tables.columntable), stmt::SQLite.Stmt, params::NamedTuple{(), Tuple{}})
    @ DBInterface ~/.julia/packages/DBInterface/qcAwq/src/DBInterface.jl:127
  [8] execute(f::Function, conn::SQLite.DB, sql::String, params::NamedTuple{(), Tuple{}})
    @ DBInterface ~/.julia/packages/DBInterface/qcAwq/src/DBInterface.jl:136
  [9] #execute#3
    @ ~/.julia/packages/DBInterface/qcAwq/src/DBInterface.jl:145 [inlined]
 [10] execute
    @ ~/.julia/packages/DBInterface/qcAwq/src/DBInterface.jl:145 [inlined]
 [11] tables (repeats 2 times)
    @ ~/.julia/packages/SQLite/P2GU2/src/SQLite.jl:660 [inlined]
 [12] top-level scope
    @ REPL[7]:1
```

Or I create the table by SQL statement, the same problem occurs.

**Comment by quinnj** [comment]
This should be fixed as of the WeakRefStrings 1.2.0 release (https://github.com/JuliaData/WeakRefStrings.jl/releases/tag/v1.2.0). Can you both share which version of WeakRefStrings you have installed and update if necessary?

**Comment by AquaIndigo** [comment]
> This should be fixed as of the WeakRefStrings 1.2.0 release (https://github.com/JuliaData/WeakRefStrings.jl/releases/tag/v1.2.0). Can you both share which version of WeakRefStrings you have installed and update if necessary?

That's perfect! And my previous solution with WeakRefStrings@1.1 is to add the method `Base.promote_type(u::Type{Union{}}, s::Type{String}) = String`.
